packages/btcindexer/src/btcblock.ts

```diff
@@ -1,19 +1,42 @@
-export interface Tx {
-	id: string;
+import { Block } from 'bitcoinjs-lib';
 
-	raw: Uint8Array; // raw tx bytes
```

Comment:
I've added it to avoid back and forth serialization when adding tx to the KV store

Reply:
ok